behave_django/runner.py

```diff
@@ -0,0 +1,62 @@
+try:
+    from django.test.runner import DiscoverRunner
+except ImportError:
+    from django.test.simple import DjangoTestSuiteRunner as DiscoverRunner
+
+from django.core.management import call_command
+
+try:
+    from django.shortcuts import resolve_url
+except ImportError:
+    # support Django 1.4, which has no resolve_url() yet
+    from django.shortcuts import redirect
+
+    resolve_url = lambda to, *args, **kwargs: \
+        redirect(to, *args, **kwargs)['Location']
+
+from behave_django.testcase import (BehaviorDrivenTestCase,
+                                    ExistingDatabaseTestCase)
+
+
+class BehaviorDrivenTestRunner(DiscoverRunner):
+    testcase_class = BehaviorDrivenTestCase
+
+    def before_scenario(self, context):
```

Comment:
I can do that. Though, I feel the `runner` module is small enough in size (53 lines fit on a single page) and easily readable (just two classes).

What I'm more concerned about for readability is the behave management command module. I find it very difficult to read the `get_behave_options()` function. The comments are an indicator for code that's (slightly) difficult to follow. But acknowledged, we're complaining on a rather high level (if you say so).